README.md

```diff
@@ -110,5 +110,37 @@ This function is deprecated. Use the `torch.nn.attention.sdpa_kernel` context ma
 Migration guide:
 Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
 
+#### torch.chain_matmul
+
+This function is deprecated in favor of `torch.linalg.multi_dot`.
```

Comment:
@kit1980  should we follow the docstring guidelines [here](https://github.com/pytorch/pytorch/wiki/Docstring-Guidelines#documenting-deprecated-methods) for deprecated functions?

Reply:
@svekars No, this is not a docstring for a deprecated function (that will be in the PyTorch repo).